src/application-development/configuration.md

```diff
@@ -0,0 +1,25 @@
+# Configuration
+
+The [esp-config] crate provides a way to manage configuration settings.
+
+## Usage
+
+The full list of available options can be found in the documentation of the `esp-*` crate you are using. For ESP32-C6's options see [configuration].
```

Comment:
> For ESP32-C6's options see [configuration].

We should mention the crate here, otherwise it sounds like all the options available for that chip - e.g. "For esp-hal on ESP32-C6 options see [configuration]`